webapp/client/cypress/integration/newsletterRegisterBox.spec.js

```diff
@@ -0,0 +1,22 @@
+describe("NewsletterRegisterBox", () => {
+  beforeEach(() => {
+    cy.visit("/unlock_code_request/step/unlock_code_success");
+  });
+
+  it("Should link to data privacy page", () => {
+    cy.get("a").contains("Datenschutzerklärung").click();
+    cy.url().should("include", "/datenschutz");
+  });
+
+  it("Should display error email empty", () => {
```

Comment:
if we have a mocked version then we can also test a submit

Reply:
See comment above